src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -713,7 +716,7 @@ export default class GroupComparisonStore {
             (this.activeGroups.isComplete && this.activeGroups.result!.length === 0 && this.tabHasBeenShown.get(GroupComparisonTab.SURVIVAL));
     }
 
-    @computed get survivalTabGrey() {
```

Comment:
why did you change it from Grey? I like Grey rather than disabled because the tab can still be clicked

Reply:
from @alisman comment https://github.com/cBioPortal/cbioportal-frontend/pull/2559#discussion_r306450700

Reply:
ah okay. I agree with @alisman 's comment. But I think disabled is the wrong word because the tab isn't really disabled. maybe survivalTabNotRendered? survivalTabUnavailable? something else? what do you think

Reply:
`survivalTabUnavailable` seems reasonable